wrappers/rust/README.md

```diff
@@ -0,0 +1,38 @@
+### Rust Bindings
+
+For convenience, we also provide rust bindings to the ICICLE library for the following primitives:
+
+- MSM
+- NTT
+    - Forward NTT
+    - Inverse NTT
+- ECNTT
+    - Forward ECNTT
+    - Inverse NTT
+- Scalar Vector Multiplication
+- Point Vector Multiplication
+
+A custom [build script][B_SCRIPT] is used to compile and link the ICICLE library. The environment variable `ARCH_TYPE` is used to determine which GPU type the library should be compiled for and it defaults to `native` when it is not set allowing the compiler to detect the installed GPU type.
+
+> NOTE: A GPU must be detectable and therefore installed if the `ARCH_TYPE` is not set.
+
+Once you have your parameters set, run:
+
+```sh
+cargo build --release
+```
+
+You'll find a release ready library at `target/release/libicicle_utils.rlib`.
```

Comment:
I think the lib name has changed

Reply:
i just didnt update it yet. please look now.